src/audits/ImageWithoutAltText.js

```diff
@@ -27,7 +28,15 @@ axs.AuditRules.addRule({
             !axs.utils.isElementOrAncestorHidden(element);
     },
     test: function(image) {
-        return (!image.hasAttribute('alt') && image.getAttribute('role') != 'presentation');
+        var imageIsPresentational = (image.hasAttribute('alt') && image.alt == '') || image.getAttribute('role') == 'presentation';
```

Comment:
Good point - we probably don't care here since it'll do the right thing with or without strict equality. It's not really necessary below either.